examples/fastyboy.rs

```diff
@@ -6,10 +6,15 @@
 //! These modifications should make the time taken to load a level in Mirror's Edge Catalyst
 //! significantly shorter.
 
-extern crate process_memory;
-#[cfg(windows)]
-extern crate winapi;
-
+mod windows {
```

Comment:
I think this should be `#[cfg(windows)]` so its not built accidentally on non-supported platforms

Reply:
Done